app/src/main/app/sql/spMinovaCheckData.sql

```diff
@@ -7,6 +7,7 @@ alter procedure dbo.spMinovaCheckData (
 	@BuildNumber int,
 	@Dataname nvarchar(255) = null
 ) with encryption as
+SET NOCOUNT ON
```

Comment:
Kannst du dokumentieren warum man das braucht?
Zumindestens in der ersten SQL-Datei?